frontend/web/components/modals/CreateFlag.js

```diff
@@ -1918,30 +1899,24 @@ const CreateFlag = class extends Component {
                                       </InfoMessage>
                                     </TabItem>
                                   )}
-                                  {isCodeReferencesEnabled && (
-                                    <TabItem tabLabel={'Code References'}>
-                                      <FeatureCodeReferencesContainer
-                                        featureId={projectFlag.id}
-                                        projectId={this.props.projectId}
-                                      />
-                                    </TabItem>
-                                  )}
-                                  {this.props.hasUnhealthyEvents && (
+                                  {
                                     <TabItem
                                       data-test='feature_health'
                                       tabLabelString='Feature Health'
                                       tabLabel={
-                                        <Row className='inline-block justify-content-center pr-1'>
-                                          Feature Health{' '}
-                                          <IonIcon
-                                            icon={warning}
-                                            style={{
-                                              color:
-                                                Constants.featureHealth
-                                                  .unhealthyColor,
-                                              marginBottom: -2,
-                                            }}
-                                          />
+                                        <Row className='d-flex justify-content-center align-items-center pr-1 gap-1'>
+                                          Feature Health
```

Comment:
I appreciate that this isn't necessarily part of the scope of this work, but this should have the 'Beta' badge next to it, like it does in the project settings menu.